recipes/s2n/all/conanfile.py

```diff
@@ -23,7 +23,7 @@ class S2n(ConanFile):
         "fPIC": True,
     }
 
-    exports_sources = "CMakeLists.txt"
+    exports_sources = "CMakeLists.txt", "patches/**"
```

Comment:
Prefer this pattern: https://github.com/conan-io/conan-center-index/blob/fe837d4775b00e8c1bf4372d0508db941aa0ba5b/recipes/aaplus/all/conanfile.py#L36-L39